File: frontend/src/services/openapi/models/ScoreboardSettingsFileModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ScoreboardMode } from './ScoreboardMode';

export type ScoreboardSettingsFileModel = {
    scoreboardMode?: ScoreboardMode;
    /**
     * Name of the currently selected division
     */
    activeDivision?: string | null;
}

File: src/Api.Models/ScoreboardSettingsFileModel.cs
namespace Api.Models
{
    public class ScoreboardSettingsFileModel
    {
        public ScoreboardMode ScoreboardMode { get; set; }

        /// <summary>
        /// Name of the currently selected division
        /// </summary>
        /// <example>Senior women</example>
        public string? ActiveDivision { get; set; }

        public int? ActiveResult { get; set; }
    }
}
